plugins/cassandra/cassandrawu.cpp

```diff
@@ -2669,7 +2669,7 @@ class CCassandraWorkUnit : public CPersistedWorkUnit
     {
     public:
         CCassandraWuGraphStats(const CCassandraWorkUnit *_parent, StatisticCreatorType _creatorType, const char * _creator, const char * _rootScope, unsigned _id)
-        : CWuGraphStats(createPTree(_rootScope), _creatorType, _creator, _rootScope, _id),
+        : CWuGraphStats(createPTree(_rootScope), _creatorType, _creator, 0, _rootScope, _id),
```

Comment:
This isn't correct.  The virtual updateStats() should have an override attribute on it - which would have caught the problem that the prototype has changed.

Reply:
override attribute added to updateStats() & prototype modified to reflect base class.